src/shipit_pulse_listener/default.nix

```diff
@@ -24,6 +24,29 @@ EOF
     '';
   });
 
+  mkBot = branch:
+    let
+      secretsKey = "repo:github.com/mozilla-releng/services:branch:" + branch;
+    in
+      mkTaskclusterHook {
+        name = "Triggers Taskcluster hooks on pulse messages";
+        owner = "babadie@mozilla.com";
+        schedule = [ "0 0 * * * *" ];  # every hour
```

Comment:
@La0 is there a reason why this needs to be recreated every hour and not running this as a service on heroku?